src/thor/map_matcher.cc

```diff
@@ -194,7 +193,7 @@ MapMatcher::FormPath(meili::MapMatcher* matcher,
     // Check if connected to prior edge
     if (prior_edge.Is_Valid() &&
         !matcher->graphreader().AreEdgesConnectedForward(prior_edge, edge_id)) {
-      disconnected_edges.emplace_back(prior_edge, edge_id);
+      disconnected_edges.insert({prior_edge, edge_id});
```

Comment:
is there actually a semantic difference here? sends rvalue ref to insert and the previous forwards the args to in place constructor.

Reply:
ah yeah derp, sorry for the confusion